scripts/inference-providers/templates/common/follow-us-button.handlebars

```diff
@@ -0,0 +1,6 @@
+<div class="flex justify-center">
+    <a href="https://huggingface.co/{{provider}}" target="_blank">
```

Comment:
actually... the provider org pages are subtly ≠ from the provider id (there are two exceptions i think)

so i think you need to maintain another mapping, or just call the template with an explicit argument